news/feature-3136.md

```diff
@@ -1,11 +1,7 @@
 `set-facility()`: add new rewrite operation to change the syslog facility
 associated with the message.
 
-<details>
-  <summary>Example config, click to expand!</summary>
```

Comment:
Why did this got removed? Too small snippet?

Reply:
Yes, I feel safer to avoid `<details>` when possible, because I do not know how will the notification email look like. This snippet seemed small enough, so I removed the details tag around it. I can put back if you want.